libnetdata/buffer/buffer.c

```diff
@@ -29,6 +29,14 @@ static inline void _buffer_overflow_check(BUFFER *b, const char *file, const cha
     }
 }
 
+//to know how we are configured at runtime
+char* buffer_mempool_status() {
```

Comment:
We are trying to define a new pattern for Netdata code, this patterns says for we use `*` near the function name instead the type.

Reply:
We said we'll first do a PR that applies the new code style to all existing code and the ask everyone writing C code to utilize the `.clang-format` we have in the repo. I say let's first concentrate on whether there's anything wrong with what -based on the benchmarks- appears to be an excellent improvement.

Reply:
agreed, that is a point of this PR at this stage. I have to go over it again myself as at this point there might be skeletons in closet. I should have more time to work on it during weekend.